segway.wdl

```diff
@@ -66,6 +66,16 @@ workflow segway {
     File segway_output_bed_ = select_first([segway_output_bed, segway_annotate.output_bed])
     File segway_params_ = select_first([segway_params, segway_annotate.segway_params])
 
+    Boolean has_segway_output_bed = defined(segway_output_bed)
```

Comment:
I don't _think_ you need this. The `select_first` in line 66 will cause the workflow to fail if either `segway_output_bed` or `segway_annotate.output_bed` don't exist.

```suggestion
```